pkgs/development/python-modules/video2midi/default.nix

```
@@ -1,24 +1,24 @@
-{ stdenv, fetchFromGitHub, python2Packages, opencv3 }:
+{ stdenv, fetchFromGitHub, pythonPackages, opencv3 }:
 
 let
-  opencv3_ = python2Packages.toPythonModule (opencv3.override {
+  opencv3_ = pythonPackages.toPythonModule (opencv3.override {
     enablePython = true;
-    pythonPackages = python2Packages;
+    pythonPackages = pythonPackages;
     enableFfmpeg = true;
   });
 in stdenv.mkDerivation rec {
   pname = "video2midi";
-  version = "0.3.7";
+  version = "0.3.9.1";
 
```

```suggestion

  format = "other";
```

Will be needed when you switch to `buildPythonApplication` since it doesn't use `setuptools` but a custom `installPhase`.

---

ref `9.14.2.2.1.1. buildPythonPackage parameters` nixpkgs manual